pkg/registry/core/pod/etcd/eviction.go

```diff
@@ -66,41 +67,53 @@ func (r *EvictionREST) Create(ctx api.Context, obj runtime.Object) (runtime.Obje
 		return nil, err
 	}
 	pod := obj.(*api.Pod)
-	pdbs, err := r.getPodDisruptionBudgets(ctx, pod)
-	if err != nil {
-		return nil, err
-	}
-
-	if len(pdbs) > 1 {
-		return &unversioned.Status{
-			Status:  unversioned.StatusFailure,
-			Message: "This pod has more than one PodDisruptionBudget, which the eviction subresource does not support.",
-			Code:    500,
-		}, nil
-	} else if len(pdbs) == 1 {
-		pdb := pdbs[0]
-		// Try to verify-and-decrement
-
-		// If it was false already, or if it becomes false during the course of our retries,
-		// raise an error marked as a 429.
-		ok, err := r.checkAndDecrement(pod.Namespace, pod.Name, pdb)
+	var rtStatus *unversioned.Status
+	err = retry.RetryOnConflict(retry.DefaultBackoff, func() error {
```

Comment:
Increasing to 10 would exponentially increase the times we wait. A request could end up waiting hours to execute.